include/flyft/mesh.h

```diff
@@ -1,6 +1,7 @@
 #ifndef FLYFT_MESH_H_
 #define FLYFT_MESH_H_
 #include "flyft/data_view.h"
+#include "flyft/boundary_type.h"
```

Comment:
Add headers alphabetically
```suggestion
#include "flyft/boundary_type.h"
#include "flyft/data_view.h"
```